js/index.js

```diff
@@ -40,3 +40,58 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// NAV Items
+const navItems = ['Services', 'Product', 'Vision', 'Features', 'About', 'Contact'];
+const navSetup = document.querySelectorAll('nav a');
+navSetup.forEach(( currentValue, index) => {
+  currentValue.textContent = siteContent.nav[`nav-item-${index + 1}`];
+});
+
+// CTA 
+let headImg = document.getElementById('cta-img');
+headImg.setAttribute('src', "img/header-img.png");
+
+let midImg = document.getElementById('middle-img');
+midImg.setAttribute('src', 'img/mid-page-accent.jpg');
+
+const mainHeader = document.querySelector('h1');
+mainHeader.textContent = 'DOM Is Awesome';
+
+const btnText = document.querySelector('.cta-text > button');
+btnText.textContent = 'Get Started!';
+
+// MAIN CONTENT
+const textContentHeading = document.querySelectorAll('.text-content h4');
+textContentHeading[0].textContent = 'Features';
+textContentHeading[1].textContent = 'About';
```

Comment:
This should all have been pulled down from the JSON like you did when setting the attributes for the `navSetup` using the `[ ]` bracket notation to access the information within.